src/AlgebraicGeometry/Schemes/ToricSchemes/ToricDivisors.jl

```diff
@@ -0,0 +1,87 @@
+########################################################################
+# This file is to allow all functionality for toric divisors on 
+# normal toric varieties also for toric schemes. 
+#
+# The functionality is simply forwarded from the `underlying_toric_variety` 
+# and for now returns an object which lives only in the toric world. 
+#
+# In the long run, we wish to eventually replace the respective return 
+# values by new objects which do contain the original toric object in the 
+# internals and implement all the toric functionality inherited from them, 
+# but, at the same time, are placed within the schemes branch of the 
+# type tree and allow to construct the geometric content of the object. 
+########################################################################
+
+function divisor_of_character(X::ToricCoveredScheme, character::Vector{T}) where {T<:Integer}
+  return divisor_of_character(underlying_toric_variety(X), character)
+end
+
+# Later, we shall return an instance of a new wrapper-type which has the 
+# full functionality of a `ToricDivisor`:
+
+#function divisor_of_character(X::ToricCoveredScheme, character::Vector{T}) where {T<:Integer}
+#  td = divisor_of_character(underlying_toric_variety(X), character)
+#  return ToricDivisorOnScheme(X, td)
+#end
```

Comment:
Just uncomment this to replace lines 15 -- 17. Then you're using the `ToricDivisorOnScheme` stuff. It's a bit messy with the order of loading, since this is in experimental, but if you manage that, then the tests in `toric_divisors.jl` still go through! At least for me.